### PR TITLE
chydube script

### PR DESCRIPTION
**Pull request by chydube**
Script written in PHP, review and merged.

### CODE REVIEW

**Comment by ajdelgados** [comment]
I see a extra dot
Pls, modify your code with following instructions https://hngi7.slack.com/archives/C014MD11S7P/p1591189194050900

**Comment by chydube** [comment]
typo fixed

File: scripts/chydube.php [hunk 1]
```diff
@@ -3,7 +3,7 @@
  //function to desplay intern detials
  function intern($name, $id, $pro_lang, $email)
  {
-     return "Hello World, this is ".$name. " with HNGi7 id ".$id. " using ".$pro_lang." for stage 2 task..".$email;
+     return "Hello World, this is ".$name. " with HNGi7 id ".$id. " using ".$pro_lang." for stage 2 task.".$email;
```
Comment (NdubuisiJr) [inline scripts/chydube.php:6]: All you need is one dot between task and email. you have two right now

Comment (chydube) [reply scripts/chydube.php:6]: The other dot is a concatenating dot ... I run a test on my local system it worked

**Pull request merged**